internal/host/refresh_status_preprocessor.go

```diff
@@ -277,6 +277,10 @@ func newValidations(v *validator) []validation {
 			id:        NoSkipMissingDisk,
 			condition: v.noSkipMissingDisk,
 		},
+		{
+			id:        NoIPCollisions,
```

Comment:
Can we include the word `Arping` in this validation name? No IP collisions sounds kinda like you're making sure all cluster hosts have a unique IP, while in reality we're doing something a little more specific - we're making sure arping doesn't indicate *some device* (as in, not necessarily a cluster host) on the network is using an IP that's being used by one of our cluster hosts

Reply:
Or maybe `NoIPCollisionsInNetwork`? Is renaming the validation in the future something expensive? Because I foresee a scenario when we switch from `arping` to some other tool (or maybe even protocol)

Reply:
Renaming in the future is expe